tests/unittests/OperatorTest.cpp

```diff
@@ -2684,6 +2684,51 @@ TEST_P(InterpAndCPU, Int8Tanh) {
   }
 }
 
+/// Verify that a quantized Log works correctly.
+TEST_P(InterpAndCPU, Int8Log) {
```

Comment:
Would it be possible to add the log as a node to an existing network? This could minimize the number of tests that we need to update in the current and next refactoring.

Reply:
I think it's important to compare the float vs. int8 results to make sure they're relatively close. The only somewhat alternative to this (other than basically replicating what's in this test in another) would be to add it to the quantizationTest's end2end test. However I feel that test is more about making sure quantization via profiling works, not numerical comparison of results. Did you have something else in mind or a different opinion on this?

Reply:
@nadavrot Let me know if you would like me to change this test.

Reply:
Sorry for the delay in response. Makes sense. Everything LGTM. Thank you for doing this work.